### DISCUSSION TITLE
Galaxy brain badge achievement bug

### DISCUSSION THREAD

**Discussion by rickstaa** [community/community#28854] [Other Feature Feedback, Questions, & Ideas] [answered]
I think a bug in the Galaxy Brain achievement badge prevents it from being shown. I checked my answered discussions (see https://github.com/search?p=1&q=commenter%3Arickstaa&type=Discussions), and I have 23 answered discussions while I only received the 2x galaxy brain badge (see https://github.com/rickstaa?achievement=galaxy-brain&tab=achievements).

#### Answered discussions
1. https://github.com/anuraghazra/github-readme-stats/discussions/1931
2. https://github.com/anuraghazra/github-readme-stats/discussions/1932
3. https://github.com/anuraghazra/github-readme-stats/discussions/1930
4. https://github.com/anuraghazra/github-readme-stats/discussions/1878
5. https://github.com/anuraghazra/github-readme-stats/discussions/1920
6. https://github.com/anuraghazra/github-readme-stats/discussions/1916
7. https://github.com/anuraghazra/github-readme-stats/discussions/1910
8. https://github.com/anuraghazra/github-readme-stats/discussions/1914
9. https://github.com/anuraghazra/github-readme-stats/discussions/1898
10. https://github.com/anuraghazra/github-readme-stats/discussions/1880
11. https://github.com/anuraghazra/github-readme-stats/discussions/1865
12. https://github.com/anuraghazra/github-readme-stats/discussions/1847
13. https://github.com/anuraghazra/github-readme-stats/discussions/1797 (NOT FIRST MARKED ANSWER)
14. https://github.com/anuraghazra/github-readme-stats/discussions/1790
15. https://github.com/anuraghazra/github-readme-stats/discussions/1742
16. https://github.com/anuraghazra/github-readme-stats/discussions/1738
17. https://github.com/anuraghazra/github-readme-stats/discussions/1716
18. https://github.com/anuraghazra/github-readme-stats/discussions/1686
29. https://github.com/anuraghazra/github-readme-stats/discussions/1682
20. https://github.com/anuraghazra/github-readme-stats/discussions/1683
21. https://github.com/anuraghazra/github-readme-stats/discussions/719
22. https://github.com/anuraghazra/github-readme-stats/discussions/1565

I checked, and I have no self-selected answers.

**Comment by melvincwng** [comment, marked as answer]
@rickstaa, try submitting a ticket to [GitHub Bug Report](https://support.github.com/contact/bug-report) over here.

Provide them a couple of working days to investigate the issue. Personally, I do notice this issue as well in some of the other users in this discussion forum, who contributed alot in answering questions as well.

Hope this helps.

**Reply by rickstaa** [reply]
@melvincwng Great, I just did that. Thanks for your answer.

**Reply by melvincwng** [reply]
@rickstaa no problem Rick, glad that I could have helped 😊

**Comment by github-staff** [comment]
We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
